MVP-UI/MVP-UI/RecipesModule/ReccipesList/RecipePresenter/Cell/ShimerViewCell.swift

```diff
@@ -0,0 +1,241 @@
+// ShimerViewCell.swift
+// Copyright © RoadMap. All rights reserved.
+
+import UIKit
+
+/// Ячейка с шимером
+final class ShimerViewCell: UITableViewCell {
+    // MARK: - Constants
+
+    private enum Constants {
+        static let gradientKey = "shimerCell"
+    }
+
+    // MARK: - VIsual Components
+
+    private let uiViewBackground: UIView = {
+        let uiView = UIView()
+        uiView.layer.cornerRadius = 12
+        uiView.translatesAutoresizingMaskIntoConstraints = false
+        return uiView
+    }()
+
+    private let recipeImageView: UIImageView = {
+        let imageView = UIImageView()
+        imageView.translatesAutoresizingMaskIntoConstraints = false
+        return imageView
+    }()
+
+    private let titleRecipe: UILabel = {
+        let label = UILabel()
+        label.translatesAutoresizingMaskIntoConstraints = false
+        return label
+    }()
```

Comment:
```suggestion
    private let titleRecipe = UILabel()
```
если настроек нет никаких, то лучше вынести

label.translatesAutoresizingMaskIntoConstraints = false. в отдельный метод
и так для всех,у кого в настройках только контрейнты